src/sentry/sentry_metrics/indexer/cache.py

```diff
@@ -38,63 +40,60 @@ def randomized_ttl(self) -> int:
         jitter = random.uniform(0, 0.25) * cache_ttl
         return int(cache_ttl + jitter)
 
-    def make_cache_key(self, key: str, cache_namespace: str) -> str:
-        hashed = md5_text(key).hexdigest()
-        return f"indexer:{self.partition_key}:org:str:{cache_namespace}:{hashed}"
+    def make_cache_key(self, key: str) -> str:
+        use_case_id, org_id, string = key.split(":", 2)
+        org_string = org_id + ":" + string
```

Comment:
do we need to split this up just to put it back together?

maybe we can just do 

```python
use_case_id, org_string = key.split(":", 1)
```

thoughts?

Reply:
that would also work I believe